scripts/Credit-Card-Slips.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+#make the logs dir if it doesn't already exist
+mkdir /mnt/alma/logs
+
+#change to the alma-scripts directory
+cd /mnt/alma/alma-scripts
+
+#install the dependencies and dev tools for them
+/usr/bin/python3.8 -m pipenv --python 3.8 install --dev
+
+#run the update, which automatically only uses the current days files 
+/usr/bin/python3.8 -m pipenv run llama cc-slips --source_email noreply@libraries.mit.edu --recipient_email ils-lib@mit.edu --recipient_email monoacq@mit.edu > /mnt/alma/logs/credit-card-slips.log 2>&1
```

Comment:
Doesn't look like we are passing the SSM parameter names here and they are required, so this script will fail. Should those params be pulling from ENV if not passed to the command?

Reply:
Forgot to update this after adding the parameters to the CLI, can't use ENV since that was the reason for adding the SSM process. Adding them into the bash script